### ISSUE TITLE
Hide 'Filter' and 'Current Map View' Boxes

### ISSUE THREAD

**Issue by kwalshe62** [umap-project/umap#1361]
**Is your feature request related to a problem? Please describe.**
_Hi, I would like to enquire if it may be possible to hide the 'Filter' and 'Current Map View' boxes, please. This would allow for more space when viewing on iPhone. Thank you._

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
![image](https://github.com/umap-project/umap/assets/135322314/69786972-3f17-4191-a1e2-a1a6b83fe331)

**Comment by kwalshe62** [comment]
Sounds like good solution.

Thank you.

**Comment by kwalshe62** [comment]
Thank you Yohan.
That’s excellent.
Katherine.

**Comment by forteller** [comment]
Hi. I came here to request to make the "Current map view" feature more visible/discoverable. Searching I found this issue, saying it was hidden just now this May. So I don't know if there's any chance to have you change your mind, but I just want to say that this is a very useful feature – I think most of the time you're most interested in the area of the map that you're looking at, and if you're interested in things other places, it's often simplest to just move the map there, I think. Showing all elements as the default can be overwhelming and unhelpful. 

Of course, that depends a lot on the type of map you're making, but I would think this is true quite often.

I would like to propose to move this switch to the right of the Filters button, something like this:

![image](https://github.com/user-attachments/assets/07d2b37b-c03b-48e4-a2b3-c893325338fa)

If this is not acceptable, I'd like to ask for a toggle in the preferences to set current map view as the default.

Thank you so much!
